<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>1047</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftVL">
      <item>
       <widget class="MemoryWidget" name="memoryWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="StateWidget" name="stateWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="HeatMapWidget" name="heatMapWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>511</width>
         <height>951</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="midVL">
        <item>
         <widget class="TraceWidget" name="listWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightVL" stretch="0,1">
      <item>
       <widget class="AsmTextEdit" name="asmTextEdit"/>
      </item>
      <item>
       <widget class="VcrWidget" name="vcrWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>349</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16000000</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_XEX"/>
  </widget>
  <action name="actionLoad_XEX">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/rsrc/icons/xex.png</normaloff>:/icon/rsrc/icons/xex.png</iconset>
   </property>
   <property name="text">
    <string>Load XEX</string>
   </property>
   <property name="iconText">
    <string>Load XEX</string>
   </property>
   <property name="toolTip">
    <string>Load XEX</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MemoryWidget</class>
   <extends>QWidget</extends>
   <header>memorywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StateWidget</class>
   <extends>QWidget</extends>
   <header>statewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HeatMapWidget</class>
   <extends>QWidget</extends>
   <header>heatmapwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VcrWidget</class>
   <extends>QWidget</extends>
   <header>vcrwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AsmTextEdit</class>
   <extends>QTextEdit</extends>
   <header>asmtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>TraceWidget</class>
   <extends>QListWidget</extends>
   <header>tracewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
